### PR TITLE
Kickoff

### PR DESCRIPTION
**Pull request by Dinuuu**
### Summary:
   <ul><li>Full engine Facebook Connect Gem Kickoff with Rspec tests and Postgres </li>
</ul>

### Trello Card:
https://trello.com/c/aOyFRjQY/1-kickoff

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+language: ruby
+rvm:
+  - 2.0.0-p598
```
Comment (epintos) [inline .travis.yml]: You should use 2.2.2

File: spec/dummy/app/assets/javascripts/application.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
+// listed below.
```
Comment (epintos) [inline spec/dummy/app/assets/javascripts/application.js]: Will you use all this?

File: spec/dummy/app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
```
Comment (epintos) [inline spec/dummy/app/assets/stylesheets/application.css]: And this?

File: spec/dummy/app/helpers/application_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module ApplicationHelper
```
Comment (epintos) [inline spec/dummy/app/helpers/application_helper.rb]: And this?

File: spec/dummy/app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Dummy</title>
```
Comment (epintos) [inline spec/dummy/app/views/layouts/application.html.erb]: Sahere

Comment (epintos) [inline spec/dummy/app/views/layouts/application.html.erb]: I mean, same here

File: spec/dummy/config/database.yml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+development: &default
+  adapter: postgresql
+  encoding: unicode
+  database: facebook_connect_development
+  pool: 5
+  username: <%= ENV.fetch('DB_USERNAME', 'facebook_connect') %>
```
Comment (epintos) [inline spec/dummy/config/database.yml]: If you are using Travis, you don't need this

File: spec/dummy/config/locales/en.yml [hunk 7]
```diff
@@ -0,0 +1,2 @@
+en:
```
Comment (epintos) [inline spec/dummy/config/locales/en.yml]: Is I18n necessary?

**Comment by epintos** [comment]
@Dinuuu I'm sure there are lots of files you don't need for this Gem

**Comment by Dinuuu** [comment]
I 've just delete some innecesary files @epintos

**Comment by epintos** [comment]
AMEENDEASTEEE

File: spec/dummy/public/404.html [hunk 8]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (epintos) [inline spec/dummy/public/404.html]: I think this is not necessary also

Comment (epintos) [inline spec/dummy/public/404.html]: and all the other errors

**Comment by epintos** [comment]
Remove the other ones and merge directly :+1:

**Pull request merged**